recipe/meta.yaml

```diff
@@ -1,29 +1,26 @@
-{% set name = "pyjanitor" %}
-{% set version = "0.1.1" %}
-{% set sha256 = "0f8aefe5933c691843b460978f1351180b51e2c12892fdc8715d688eda9b402b" %}
+{% set version = "0.1.2" %}
 
 package:
-  name: {{ name|lower }}
+  name: pyjanitor
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/p/pyjanitor/pyjanitor-{{ version }}.tar.gz
+  sha256: f32297e8a412dee0edf4397182443fdbb96139466fd055c0f6f904a61ab98a8b
 
 build:
   number: 0
   script: python -m pip install --no-deps --ignore-installed .
-  skip: True  # [py<36]
 
 requirements:
   build:
-    - python
+    - python >=3.6
```

Comment:
@ericmjl by not using the pre-selector and using the constraint here instead we can make the recipe noarch and build it faster.